StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/VerticalSavedPaymentMethodsViewControllerSnapshotTests.swift

```diff
@@ -28,14 +28,16 @@ final class VerticalSavedPaymentMethodsViewControllerSnapshotTests: STPSnapshotT
         var configuration = PaymentSheet.Configuration()
         configuration.appearance = appearance
         let sut = VerticalSavedPaymentMethodsViewController(configuration: configuration, paymentMethods: generatePaymentMethods())
+        let bottomSheet = BottomSheetViewController(contentViewController: sut, appearance: appearance, isTestMode: true, didCancelNative3DS2: {})
+        bottomSheet.view.autosizeHeight(width: 375)
+
         let testWindow = UIWindow(frame: CGRect(x: 0, y: 0, width: 428, height: 500))
         testWindow.isHidden = false
         if darkMode {
             testWindow.overrideUserInterfaceStyle = .dark
         }
-        testWindow.rootViewController = sut
-        sut.view.autosizeHeight(width: 375)
-        STPSnapshotVerifyView(sut.view)
+        testWindow.rootViewController = bottomSheet
+        STPSnapshotVerifyView(bottomSheet.view)
```

Comment:
Updated this so the window size is the height of the VC + the BottomSheet. For some reason even w/ a content view controller the height of the BottomSheet is always just the height of the nav bar (42). FWIW, the testing framework uses the [screen bounds](https://github.com/uber/ios-snapshot-test-case/blob/57b023c8bb3df361e2fae01532cd066ec0b65d2e/src/iOSSnapshotTestCase/SwiftSupport.swift#L23) as the VC bounds when generating it's view, but instead we use `FBSnapshotVerifyView`.

https://github.com/uber/ios-snapshot-test-case/blob/57b023c8bb3df361e2fae01532cd066ec0b65d2e/src/iOSSnapshotTestCase/SwiftSupport.swift#L23

Reply:
Hmm weird. We should have one simple way to snapshot test these bottom sheet content VCs, we're doing this many times and shouldn't be re-solving it each time. 

Our content VCs are special in that they are not presented full screen like normal VCs so I think it's important to simulate using the height of the VC as it would appear in the bottom sheet.  Like if our VC has a bug that causes it to be too tall when presented in the sheet, that would be good to catch in the snapshot test.

Reply:
Agreed, I'll look at this closer in a following PR